### PR TITLE
At/cb 148 create a mock for uncovered portfolio

### PR DESCRIPTION
**Pull request by Angelbt12**
Create a mock for "uncovered portfolio"
image

### CODE REVIEW

**Review by diegoAndres01 — APPROVED** [review]

File: src/mocks/creditProfileInfo/creditBehavior.mock.ts [hunk 1]
```diff
@@ -0,0 +1,66 @@
+export const credit_behaviorInfo = [
+  {
+    credit_request_id: "999990",
+    Credit_behavior: {
```
Comment (carin91) [inline src/mocks/creditProfileInfo/creditBehavior.mock.ts:4]: all of the fields should be in lowercase

File: src/mocks/creditProfileInfo/creditBehavior.mock.ts [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+        core_risk_score: 250 ,
```
Comment (carin91) [inline src/mocks/creditProfileInfo/creditBehavior.mock.ts:5]: The information must include a date.

File: src/mocks/creditProfileInfo/creditBehavior.mock.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
+export const credit_behaviorInfo = [
```
Comment (carin91) [inline src/mocks/creditProfileInfo/creditBehavior.mock.ts:1]: The task considers rendering the information on every fieldset based on the mock.
image
As an example, you could review the code which controls the risk scoring rendering
image

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/mocks/creditProfileInfo/creditBehavior.mock.ts (hunk 1 above)
Comment (Angelbt12) [reply src/mocks/creditProfileInfo/creditBehavior.mock.ts:4]: ok

File: src/mocks/creditProfileInfo/creditBehavior.mock.ts (hunk 2 above)
Comment (Angelbt12) [reply src/mocks/creditProfileInfo/creditBehavior.mock.ts:5]: ok

File: src/mocks/creditProfileInfo/paymentCapacity.mock.ts [hunk 4]
```diff
@@ -0,0 +1,74 @@
+export const payment_capacityInfo = [
```
Comment (carin91) [inline src/mocks/creditProfileInfo/paymentCapacity.mock.ts:1]: adjust the spelling to payment_capacity

File: src/mocks/creditProfileInfo/creditBehavior.mock.ts [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+  {
+    credit_request_id: "999990",
+    credit_behavior: {
+        core_risk_score: 250 ,
+        central_risk_scor_date: "ago/31/2023",
```
Comment (carin91) [inline src/mocks/creditProfileInfo/creditBehavior.mock.ts:6]: adjust the spelling: central_risk_score_date

File: src/mocks/creditProfileInfo/creditBehavior.mock.ts (hunk 3 above)
Comment (carin91) [reply src/mocks/creditProfileInfo/creditBehavior.mock.ts:1]: This activity is still pending

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/mocks/creditProfileInfo/creditBehavior.mock.ts (hunk 5 above)
Comment (Angelbt12) [reply src/mocks/creditProfileInfo/creditBehavior.mock.ts:6]: ok

File: src/mocks/creditProfileInfo/creditBehavior.mock.ts (hunk 3 above)
Comment (carin91) [reply src/mocks/creditProfileInfo/creditBehavior.mock.ts]: This activity is still pending

**Review by carin91 — APPROVED** [review]

**Pull request merged**